src/server/pmix_server_get.c

```diff
@@ -218,11 +218,12 @@ pmix_status_t pmix_server_get(pmix_buffer_t *buf,
     /* if the rank is wildcard, then they are asking for the job-level
      * info for this nspace - provide it */
     if (PMIX_RANK_WILDCARD == rank) {
+        pmix_buffer_t **bptr = &nptr->server->job_info; 
         PMIX_CONSTRUCT(&pbkt, pmix_buffer_t);
         pmix_bfrop.pack(&pbkt, &rank, 1, PMIX_PROC_RANK);
         /* the client is expecting this to arrive as a byte object
          * containing a buffer, so package it accordingly */
-        pmix_bfrop.pack(&pbkt, &nptr->server->job_info, 1, PMIX_BUFFER);
+        pmix_bfrop.pack(&pbkt, &bptr, 1, PMIX_BUFFER);
```

Comment:
This doesn't look correct - you are passing a triple-star to a pmix_buffer object?? Shouldn't it be a double-star?